src/components/TruckSizeSelect/TruckSizeSelect.css

```diff
@@ -19,8 +19,10 @@
 
 .truck-size-container {
     display: flex;
+    max-width: 600px;
     flex-direction: row;
     justify-content: space-around;
     align-items: center;
+    margin-bottom: 2rem;

```

Comment:
BTW I find it pretty useful to use bootstrap classes then you have a better idea what is happening before going to this extra file.

`<div className="mb-5 d-flex flex-column justify-content-around align-items-center">` tells me a lot without going to a separate file
`<div className="truck-size-container">` this tells me nothing without going to a separate file.

Reply:
Ah, yes that does seem better. I have not used bootstrap much before, but this seems like a good opportunity to learn it ^^

Reply:
Changed to bootstrap on a couple of simple places